src/components/PeopleDropdown/PeopleDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import debounce from 'lodash.debounce';
+import React, { useCallback } from 'react';
+
+interface Props {
+  query: string;
+  isHide: boolean;
+  delay: {
+    mainDelay: number,
+    blurDelay: number,
+  };
+  setQuery: (value: string) => void;
+  setIsHide: (value: boolean) => void;
+  applyQuery: (value: string) => void;
+}
+
+export const PeopleDropdown: React.FC<Props> = ({
+  query,
+  isHide,
+  delay,
+  setIsHide,
+  setQuery,
+  applyQuery,
+}) => {
+  const handleClickHide = () => {
+    return isHide ? setIsHide(false) : setIsHide(true);
+  };
```

Comment:
I think you can write ` setIsHide(!isHide)`

Reply:
do you need return this value?

Reply:
no